nuxt.config.ts

```diff
@@ -104,6 +104,29 @@ export default defineNuxtConfig({
     },
   },
 
+  routeRules: {
+    '/api/store/products': {
+      security: {
+        corsHandler: {
+          origin: '^https?:\\/\\/([a-zA-Z0-9-]+\\.)?3ook\\.com$',
```

Comment:
no need for double backslashes in string?

Reply:
seems this string is directly passed into `new RegExp`